cmd/context/show.go

```diff
@@ -17,21 +17,42 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"reflect"
+	"strings"
 
 	"github.com/okteto/okteto/cmd/utils"
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 	"github.com/okteto/okteto/pkg/okteto"
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
 )
 
+const hintUrl = "https://okteto.com/docs/reference/cli/#show"
+
+func printContextMember(ctx *okteto.OktetoContext, key string) error {
+	element := reflect.ValueOf(ctx).Elem()
+	for i := 0; i < element.NumField(); i++ {
+		if strings.EqualFold(element.Type().Field(i).Name, key) {
+			memberValue := fmt.Sprintf("%v", element.Field(i).Interface())
+			oktetoLog.Println(memberValue)
```

Comment:
I think it'd be great if the new code we add is fully testable, this includes printing. 
We have 2 options here:
- Add the printing function as a parameter, something like `func printContextMember(ctx *okteto.OktetoContext, key string, print func(...interface{})) error`
- Return `(string, error)` and let the caller do the printing and error handling